### PR TITLE
Feature/d3 asim 2895

### PR DESCRIPTION
**Pull request by faizan2590**

### CODE REVIEW

File: src/d3a/models/area/stats.py [hunk 1]
```diff
@@ -131,20 +130,18 @@ def current_market(self):
...
-                                 f"the format '{DATE_TIME_FORMAT}'"}
-            out_dict[time_slot_str] = copy(self.rate_stats_market.get(
-                time_slot, default_trade_stats_dict))
-            out_dict[time_slot_str]["market_bill"] = self._get_market_bills(time_slot)
-            if dso:
-                out_dict[time_slot_str]["area_throughput"] = \
-                    self._get_market_area_throughput(time_slot)
+        if self.current_market is None:
+            return out_dict
+        current_market_time_slot_str = self.current_market.time_slot_str
```
Comment (hannesdiedrich) [inline src/d3a/models/area/stats.py:137]: I think you can skip this, the new name is not really shorter than the direct variable name :-)

**Review by hannesdiedrich — APPROVED** [review]
One minor comment, LGTM
(please let Chris test this, if not already done, thanks!)

**Review by spyrostz — DISMISSED** [review]
LGTM

**Review by spyrostz — APPROVED** [review]
LGTM

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Pull request merged**